Gemfile

```diff
@@ -18,4 +18,5 @@ group :test do
   gem 'minitest-reporters', require: false
   gem 'fakefs', require: false
   gem 'webmock', require: false
+  gem 'timecop', require: false
```

Comment:
Maybe we could inject the time into `ShopifyCli::Core::Monorail` rather than adding this dependency?

Reply:
eh, I don't mind adding test dependencies and I don't love adding parameters etc for testing purposes only. (although I am guilty of doing it in other places in this codebase). I could go either way on this.